2-ui/4-forms-controls/1-form-elements/article.md

```diff
@@ -289,9 +289,9 @@ Form navigation:
 `element.form`
 : Elements reference their form in the `form` property.
 
-Value is available as `input.value`, `textarea.value`, `select.value` etc, or `input.checked` for checkboxes and radio buttons.
+Value is available as `input.value`, `textarea.value`, `select.value`, etc. (or `input.checked` for checkboxes and radio buttons).
```

Comment:
Are you sure the dot is correct at the end? Doesn't look so.

Reply:
I'll rewrite...  Just a moment.

Reply:
Rewritten with less weird punctuation at the endpoints. :)